prometheus/postgres-exporter/businessInteligence.yml

```diff
@@ -0,0 +1,31 @@
+chamber_of_deputies_reimbursement:
+  query: "select distinct id, year, case when month = 1 then 'JANUARY' when month = 2 then 'FEBRUARY' when month = 3 then 'MARCH' when month = 4 then 'APRIL' when month = 5 then 'MAY' when month = 6 then 'JUNE' when month = 7 then 'JULY' when month = 8 then 'AUGUST' when month = 9 then 'SEPTEMBER' when month = 10 then 'OCTOBER' when month = 11 then 'NOVEMBER' when month = 12 then 'DECEMBER' end as month , state, party, subquota_description, supplier, congressperson_name, total_net_value from chamber_of_deputies_reimbursement"
+  metrics:
+    - id:
+        usage: "LABEL"
+        description: "Reimbursement ID"
```

Comment:
Not sure how this is going to figure in the UI, but this is useless from the UX perspective.

This ID is our Postgres-attributed unique identifier, and not the proper ID attributed by the Chamber of Deputies (which is not unique and is the `document_id`).

Reply:
It's actually kind useless, but not really

Similar to a relational database, prometheus can't store metrics with same name and labels(Relational dbs can't have duplicated primary keys).

All the registries queried by this query will have the same metric name (chamber_of_deputies_reimbursement_total_net_value), and the labels will vary according with their columns values....

I've noticed that if we don't query that ID column, we will have duplicated metrics with the exactly same labels, which will cause an error and we wouldn't be able to collect those metrics.

In short, we need to collect that ID just to avoid some errors, but its not used for anything

Reply:
Got it! Many thanks for the explanation. Maybe we could just rename the label so whoever uses it knows it's an id attributed by Serenata and not by the Chamber of Deputies. Maybe _Jarbas ID_ or a longer version _ID attributed to the reimbursement by in Serenata_…